src/org/openlcb/cdi/swing/CdiPanel.java

```diff
@@ -250,7 +251,7 @@ DisplayPane createGroupPane(CdiRep.Group item, long origin, int space) {
             }
             JPanel currentPane = this;
             JTabbedPane tabbedPane = new JTabbedPane();
-            currentPane.add(tabbedPane);
+            tabbedPane.setTabLayoutPolicy(JTabbedPane.SCROLL_TAB_LAYOUT);
```

Comment:
I actually liked the wrapping tabs better; the scroll makes it painfully slow to get to a large count on a small screen.